collectfast/tests/utils.py

```diff
@@ -2,14 +2,18 @@
 import os
 import pathlib
 import random
-import unittest
+import unittest.mock
 import uuid
 from typing import Any
 from typing import Callable
+from typing import Dict
 from typing import Type
 from typing import TypeVar
+from typing import Union
 from typing import cast
 
+import boto3  # type: ignore
+import moto  # type: ignore
```

Comment:
Let's install types for boto3, and [ignore moto in setup.cfg](https://github.com/antonagestam/collectfast/blob/master/setup.cfg#L82).

Related: pytest should be possible to remove from setup.cfg since it has gained typing support.

Reply:
Sounds great.